modules/EPMC.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+import logging
+
+import pyspark
+from pyspark.sql.types import StringType
+from pyspark.sql import SparkSession 
+import pyspark.sql.functions as pf
+from pyspark import SparkContext
+from pyspark.conf import SparkConf
+
+
+# The following target labels are excluded as they were grounded to too many target Ids
+EXCLUDED_TARGET_TERMS = ['TEC', 'TECS', 'Tec', 'tec', '\'', '(', ')', '-', '-S', 'S', 'S-', 'SS', 'SSS',
+         'Ss', 'Ss-', 's', 's-', 'ss', 'U3', 'U6', 'u6', 'SNORA70', 'U2', 'U8']
+
+def main(cooccurrenceFile, outputFile, local=False):
+
+    ##
+    ## Initialize spark session
+    ##
```

Comment:
PEP8 says [block comments](https://www.python.org/dev/peps/pep-0008/#block-comments) should start with a single '#' followed by a single space:

```suggestion
    # Initialize spark session:
```